src/Common/test-runtime/project.json

```diff
@@ -30,7 +30,7 @@
     "System.Net.WebSockets": "4.0.0-rc3-23809",
     "System.Net.WebSockets.Client": "4.0.0-rc3-23809",
     "System.ObjectModel": "4.0.0",
-    "System.Reflection": "4.0.10",
+    "System.Reflection": "4.0.0",
```

Comment:
Since 4.0.10 system.reflection is also stable version, I think it's ok to stay with it. Similarly for 4.0.20 system.runtime. Do we have a build issue if we stick with these versions?